### PR TITLE
CITATION.cff: update with contributors

### PR DESCRIPTION
**Pull request by rouault**
@dbaston @craigds @miurahr You deserve to be in the CITATION.cff file. Let me know if you're OK with this. Please feel free to add your orcid ID and set/change your affiliation if needed

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -59,6 +59,15 @@ authors:
     family-names: Dawson
     affiliation: North Road
     orcid: https://orcid.org/0000-0001-9812-7584
+  - given-names: Daniel
+    family-names: Baston
+    affiliation: ISciences, L.L.C.
```
Comment (dbaston) [inline CITATION.cff:64]:
```suggestion
    affiliation: ISciences, L.L.C.
    orcid: https://orcid.org/0000-0002-2195-1716
```

**Review by craigds — APPROVED** [review]
Thanks Even!

File: CITATION.cff [hunk 2]
```diff
@@ -59,6 +59,16 @@ authors:
...
+    orcid: https://orcid.org/0000-0002-2195-1716
+  - given-names: Craig
+    family-names: de Stigter
+    affiliation: Koordinates
+  - given-names: Hiroshi
+    family-names: Miura
```
Comment (miurahr) [inline CITATION.cff:70]:
```suggestion
  - given-names: Hiroshi
    family-names: Miura
    orcid: https://orcid.org/0000-0001-7754-0433
```

**Comment by miurahr** [comment]
Thanks.

**Pull request merged**